osu.Game.Tests/Visual/SongSelectV2/TestSceneBeatmapCarouselScrolling.cs

```diff
@@ -45,11 +41,35 @@ public void TestScrollPositionMaintainedOnAddSecondSelected()
         }
 
         [Test]
-        public void TestScrollPositionMaintainedOnAddLastSelected()
+        public void TestScrollPositionMaintainedOnAdd_SecondSelected_WithUserScroll()
```

Comment:
What's with the naming on this and the other test above? They say "maintained on add" but where's the add? I only see a `RemoveFirstBeatmap()` call...

Reply:
No idea, renamed.